test-cases/tests/products/h21b-verify-all-products-using-the-workloadwebapp.md

```diff
@@ -30,7 +30,7 @@ The [workload-web-app](https://github.com/integr8ly/workload-web-app) will:
    IMPORTANT. Make sure that you don't run `make local/deploy` again, as this will break the monitoring dashboard. If **workload-web-app** namespace exists in the cluster, the workload-web-app has already been deployed.
 
    ```bash
-    export GRAFANA_DASHBOARD=true RHOAM=true
```

Comment:
I think this should still have RHOAM=true?

Reply:
No, it shouldn't. For deploying workload web app you no longer need to specify RHOAM env variable: https://github.com/integr8ly/workload-web-app#deploying-the-application-on-the-rhoam-cluster. It was previously changed in this pr https://github.com/integr8ly/workload-web-app/pull/43.

Reply:
In that case there are a couple of places where you need to delete the RHOAM env var

Reply:
like for example in the n10 test, there are a couple others too

Reply:
Agree, thanks for spotting it